dev-db/postgresql/postgresql-9.6.10.recipe

```diff
@@ -0,0 +1,160 @@
+	lib:libecpg$secondaryArchSuffix = $portVersion compat >= 9
+	lib:libecpg_compat$secondaryArchSuffix = $portVersion compat >= 9
+	lib:libpgtypes$secondaryArchSuffix = $portVersion compat >= 9
+	lib:libpq$secondaryArchSuffix = $portVersion compat >= 9
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libreadline$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_server="
+	cmd:clusterdb$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:createdb$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:createlang$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:createuser$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:dropdb$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:droplang$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:dropuser$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:ecpg$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:initdb$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_archivecleanup$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_basebackup$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_config$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_controldata$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_ctl$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_dump$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_dumpall$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_isready$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_receivexlog$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_resetxlog$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_recvlogical$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_restore$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_rewind$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_test_fsync$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_test_timing$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_upgrade$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_xlogdump$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pgbench$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:postgres$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:postmaster$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:reindexdb$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:vacuumdb$secondaryArchSuffix = $portVersion compat >= 9
+	"
+REQUIRES_server="
+	haiku$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	postgresql${secondaryArchSuffix} == $portVersion base
+	"
+
+PROVIDES_devel="
+	postgresql${secondaryArchSuffix}_devel = $portVersion
+	devel:libecpg$secondaryArchSuffix = $portVersion
+	devel:libecpg_compat$secondaryArchSuffix = $portVersion
+	devel:libpgcommon$secondaryArchSuffix = $portVersion
+	devel:libpgport$secondaryArchSuffix = $portVersion
+	devel:libpgtypes$secondaryArchSuffix = $portVersion
+	devel:libpq$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	postgresql${secondaryArchSuffix} == $portVersion base
```

Comment:
Suggestion:
```diff
 REQUIRES_devel="
-	postgresql${secondaryArchSuffix} == $portVersion base
+	postgresql$secondaryArchSuffix == $portVersion base
 	postgresql${secondaryArchSuffix}_server == $portVersion
 	"
```
Hmm, you are right. It looks slightly nicer with the curly braces.